exercise/templates/exercise/exercise_base.html

```diff
@@ -4,7 +4,7 @@
 {% load exercise %}
 {% load apps %}
 
-{% block title %}{{ current.name }} | {{ block.super }}{% endblock %}
+{% block title %}{{ current.name|parse_localization }} | {{ block.super }}{% endblock %}
```

Comment:
Need to use translation block with variable and run the filter there

Reply:
I don't understand what the problem here is, it seems to be working fine?

Reply:
I see. Django works interestingly on that matter, so it really does work. Django strores everything inside `{{}}` as the "key" (think as variable) and uses that with `%` python format to expand the variables in translation string. Only downside from current is that the translation contains `current.name|parse_localization` instead of `name` for example. Thus in real life, it could be obscuring detail for the translator.

Relevant parts in django source:
 * store vars: https://github.com/django/django/blob/master/django/templatetags/i18n.py#L119
 * string format: https://github.com/django/django/blob/master/django/templatetags/i18n.py#L164

So in short, current code is fine and ok.